### PR TITLE
The -o option can be used multiple times No.2

### PR DESCRIPTION
**Pull request by milo**
Alternative approach to #310.

### CODE REVIEW

**Comment by dg** [comment]
`stdout` -> `stdoutFormat` :+1: 

`options['-o']` -> `$this->outputs` hmmm, perhaps unnecessary internal dependency…

I compared it on GitHub in phone and it is quite possible that I missed the other differences.

File: src/Runner/CliTester.php [hunk 1]
```diff
@@ -101,21 +107,23 @@ private function loadOptions()
...
+    -c <path>                Look for php.ini file (or look in directory) <path>.
+    -l | --log <path>        Write log to file <path>.
+    -d <key=value>...        Define INI entry 'key' with value 'val'.
+    -s                       Show information about skipped tests.
+    --stop-on-fail           Stop execution upon the first failure.
+    -j <num>                 Run <num> jobs in parallel (default: 8).
+    -o <console|tap|junit|log|none>
+                             Specify one or more output formats with optional file name.
+                             (e.g. -o junit:output.xml)
+    -w | --watch <path>      Watch directory.
```
Comment (dg) [inline src/Runner/CliTester.php]: What about `-o <console|tap|junit|log|none>  (e.g. -o junit:output.xml)` to save one row?

**Comment by milo** [comment]
> options['-o'] -> $this->outputs hmmm, perhaps unnecessary internal dependency…

Yes. That's part I don't like too but I cannot figure how to solve it in other way. Imagine I will call `-o console -o tap:output.tap -o junit:output.xml`, I would like to:
- keep ENUM check in CommandLine. So value `tap:file.name` has to be modified before this check (NORMALIZER do it) and `file.name` has to be kept somewhere for OutputHandler's constructor. The `$this->outputs` is used for this. I cannot figure how to keep `file.name` inside `$options['-o']` without hidden functionality.

- prevent more outputs into one file

File: src/Runner/CliTester.php (hunk 1 above)
Comment (milo) [inline src/Runner/CliTester.php]: I'll try that.

Comment (milo) [inline src/Runner/CliTester.php]: IMHO, it looks strange when not aligned.

**Comment by milo** [comment]
@dg Updated

**Comment by TomasVotruba** [comment]
What needs to be done here?

**Comment by milo** [comment]
Not sure. But I'm not satisfied with an implementation.

**Comment by milo** [comment]
That's what I'm nor sure about. The part with `NORMALIZER` probably. The main purpose for this PR is to allow pass multiple output handlers by CLI arguments. Propose is `-o format:file.ext`. Maybe another syntax would be better and more straightfoward for implementation.

**Pull request merged**